arm/Microsoft.Insights/components/readme.md

```diff
@@ -64,13 +64,13 @@ Tag names and tag values can be provided as needed. A tag can be left without a
 
 ## Outputs
 
-| Output Name | Type |
-| :-- | :-- |
-| `appInsightsAppId` | string |
-| `appInsightsKey` | string |
-| `appInsightsName` | string |
-| `appInsightsResourceGroup` | string |
-| `appInsightsResourceId` | string |
+| Output Name | Type | Description |
+| :-- | :-- | :-- |
+| `appInsightsAppId` | string | The application ID of the application insights component |
+| `appInsightsKey` | string | The instrumentation key of the application insights component |
+| `appInsightsName` | string | The name of the application insights component |
+| `appInsightsResourceGroup` | string | The resource group the application insights component was deployed into |
+| `appInsightsResourceId` | string | The resource ID of the application insights component |
```

Comment:
To be updated accordingly after removing additional outputs

Reply:
done